dirty_cat/test/test_datetime_encoder.py

```diff
@@ -0,0 +1,183 @@
+import pandas as pd
+from dirty_cat.datetime_encoder import DatetimeEncoder
+import numpy as np
+
+
+def get_date_array():
+    return np.array([pd.to_datetime(["2020-01-01", "2020-01-02", "2020-01-03"]),
+                  pd.to_datetime(["2021-02-03", "2020-02-04", "2021-02-05"]),
+                  pd.to_datetime(["2022-01-01", "2020-12-25", "2022-01-03"]),
+                  pd.to_datetime(["2023-02-03", "2020-02-04", "2023-02-05"])])
```

Comment:
In terms of indentation, it would be great if you indented a bit more this list, to align the "pd" to the above "pd" on the first line (comment also applies below in a few places)